src/service/chat.js

```diff
@@ -191,7 +193,8 @@ const moveChatToAnon = async ({
       anon_user_info_emoji_name,
       oldChannelId,
       targetUserId,
-      source
+      source,
+      context: 'chat'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding the `context` parameter to the request payload in the `moveChatToAnon` function call is a good change. However, it's important to ensure that the server-side code handling this request is also updated to handle this new parameter. If not, it could lead to unexpected behavior.

```diff
       anon_user_info_emoji_name,
       oldChannelId,
       targetUserId,
       source,
+      context: 'chat'
     });
     if (response.status === 200) {
       return Promise.resolve(response.data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->